### ISSUE TITLE
Add PCLinuxOS download instructions

### ISSUE THREAD

**Issue by nurupo** [Tox/tox.chat#161]
We have received a request from @marcin-82, the package maintainer of `qtox` and `toxcore` packages for [PCLinuxOS](https://distrowatch.com/table.php?distribution=pclinuxos) in main/official repository of distribution. You can see them in [this package repository mirror](http://ftp.nluug.nl/os/Linux/distr/pclinuxos/pclinuxos/srpms/SRPMS.pclos/).

Here are the instructions the maintainer has provided:

> You can install QTox on PCLinuxOS by two ways:
> I Synaptic package manager
> 1. Run Synaptic package manager and click "Refresh" to update
> repository indexes.
> 2. Search for "qtox" package and mark it for installation.
> 3. Click "Apply" and confirm installation.
> 4. Program will be available under "Internet" category in applications menu.
> 
> II Terminal
> 1. Run as root these two commands: 
> apt-get update
> apt-get install qtox
> 
> I'd like to format apt-get commands similar to BBCode - [code]text[/code] tags.
> 
> Here is logo of PCLinxuOS. I have approval/consent from Textar (Bill
> Reynolds - owner of PCLinuxOS trademark) to use it:
> https://pbs.twimg.com/profile_images/421896334/Aqua.png

**Comment by tox-user** [comment]
@nurupo I'm interested in working on this. What am I supposed to do with the image? Downloads page uses black and white icons for distro logos.

**Comment by nurupo** [comment]
Yeah, I'm not sure. We want to add it to https://tox.chat/download.html#gnulinux, but like you have said, the icon would be out of place.

**Comment by tox-user** [comment]
I made the logo gray to make it fit with other icons. Can I somehow add it to the font and use class icon-pclinuxos in css?

**Comment by nurupo** [comment]
Sorry, I'm not very familiar with web fonts and web technologies in general. I'm not a web developer, I'm more of C/C++ developer, and it has been over a year since this website had one maintaining it. I mostly just keep the links and the information up to date.

**Comment by robinlinden** [comment]
@tox-user https://glyphter.com/ was used to create the last font set, but I think you'd have to recreate the entire font if you want to add your icon to it.
